extensions/gdx-tools/src/com/badlogic/gdx/tools/etc1/AtlasETC1CompressionResult.java

```diff
@@ -0,0 +1,59 @@
+
+package com.badlogic.gdx.tools.etc1;
+
+import java.util.ArrayList;
+
+public class AtlasETC1CompressionResult {
+
+	static class TextureFileRenaming {
+
+		private String old_page_file_name;
+		private String new_page_file_name;
+
+		public TextureFileRenaming (String old_page_file_name, String new_page_file_name) {
+			this.old_page_file_name = old_page_file_name;
+			this.new_page_file_name = new_page_file_name;
+		}
+
+		@Override
+		public String toString () {
+			return "" + old_page_file_name + " :-> " + new_page_file_name + "";

```

Comment:
I thought it was left in because the compiler will fix it anyway? Anyway, please don't add extraneous things on the basis that someone might want to format it in a specific way in the future. Make the code clean and easy to read by default...  if someone wants to add more formatting, they will come along and do it when they choose.